### PR TITLE
Create Api-Final Project.md

### PR DESCRIPTION
**Pull request by Nitrogend**
Please check my GET and let me know if I am on point or what I need to fix.

### CODE REVIEW

File: student-work/chana-willinger/Api-Final Project.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## GET/tableNo
+
+This is the call sent when the server wants to get the bill. The GET request includes the table number. Take out orders are table 99. When this is printed, the customer can pay the bill.
```
Comment (lsnovich) [inline student-work/chana-willinger/Api-Final Project.md:3]:
```suggestion
This is the call sent when the patron wants to get their bill. The GET request includes the table number. Take out orders are table 99. When this is printed, the customer can pay the bill.
```

File: student-work/chana-willinger/Api-Final Project.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Code Example
+
+curl -X GET "http://URL/tableNo?id=99"
```
Comment (lsnovich) [inline student-work/chana-willinger/Api-Final Project.md:7]:
```suggestion
```curl
      curl -X GET "http://URL/tableNo?id=99"
```      
```

File: student-work/chana-willinger/Api-Final Project.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+         "type":"burgerMeal",
+         "Cost":10.99
+      }
+   },
+   "Item2":{
+      "ItemOrdered":{
+         "type":"salad",
+         "Cost":9.50
+      }
+   }
```
Comment (lsnovich) [inline student-work/chana-willinger/Api-Final Project.md:28]:
```suggestion
   };
```

**Review by lsnovich — COMMENTED** [review]
Something is off with the code, which is why it is red.

**Review by Laura-Novich-OBW — APPROVED** [review]
Nice job!!

**Pull request merged**